.github/workflows/codestyle.yml

```
@@ -108,20 +108,32 @@ jobs:
             target
           key: v4-${{ runner.os }}-cargo-${{ hashFiles('./Cargo.lock') }}-rust
 
-      # https://github.com/facebookincubator/cargo-guppy/tree/main/tools/cargo-hakari#2-keep-the-workspace-hack-up-to-date-in-ci
-      - name: Check every project module is covered by Hakari
-        run: |
-          cargo install cargo-hakari
-          cargo hakari generate --diff  # workspace-hack Cargo.toml is up-to-date
-          cargo hakari manage-deps --dry-run  # all workspace crates depend on workspace-hack
-        shell: bash -euxo pipefail {0}
-
       - name: Run cargo clippy
         run: ./run_clippy.sh
 
       - name: Ensure all project builds
         run: cargo build --locked --all --all-targets
 
+  check-rust-dependencies:
+    runs-on: dev
+    container:
+      image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/rust:pinned
+      options: --init
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```

I don't really know the difference, but sometimes we use v3